ocfweb/stats/templates/stats/accounts.html

```diff
@@ -1,14 +1,94 @@
 {% extends 'base.html' %}
+{% load common %}
 {% load stats %}
 
 {% block content %}
     <div class="ocf-content-block">
         {% stats_navbar %}
-        <p>
-            <img class="img-responsive" src="{% url 'cumulative_accounts_graph' %}" />
-        </p>
-        <p>
-            <img class="img-responsive" src="{% url 'cumulative_group_accounts_graph' %}" />
-        </p>
+    <div id="total"></div>
+    <div id="group"></div>
     </div>
 {% endblock %}
+
+{% block inline_js %}
+    <script src="https://code.highcharts.com/5.0.7/highcharts.js"></script>
+    <script>
+        $(document).ready(function () {
+            Highcharts.setOptions({
+                global: {
+                    useUTC: false
+                }
+            });
```

Comment:
I think having a more standardized template for this instead of duplicating the same first part across multiple pages (like in the pages printed template I've linked below) would be good, but that's maybe more of an enhancement after this PR's changes. If you'd like to do it then I think that would make this a bit cleaner though:

https://github.com/ocf/ocfweb/blob/96216ddd76631356d9057fd54169d463941c80a5/ocfweb/stats/templates/stats/printing/pages-printed.html#L30-L74